diarize.py

```diff
@@ -56,7 +66,10 @@
 
 
 # Run on GPU with FP16
-whisper_model = WhisperModel(args.model_name, device="cuda", compute_type="float16")
+if args.device == "cuda":
+    whisper_model = WhisperModel(args.model_name, device="cuda", compute_type="float16")
+elif args.device == "cpu":
+    whisper_model = WhisperModel(args.model_name, device="cpu", compute_type="int8")
```

Comment:
I suggest you create a dict for the compute dtype for each device instead of having an if else